deps/rabbit/test/amqp_client_SUITE.erl

```diff
@@ -5621,6 +5622,28 @@ tcp_back_pressure_rabbitmq_internal_flow(QType, Config) ->
     ok = end_session_sync(Session),
     ok = amqp10_client:close_connection(Connection).
 
+session_max(Config) ->
+    App = rabbit,
+    Par = ?FUNCTION_NAME,
```

Comment:
Should we really depend on `?FUNCTION_NAME` here? Test functions do get renamed sometimes :)

Reply:
My intent was that the test case tests the config value its named after: `session_max`.
I removed `?FUNCTION_NAME` 👍